opentech/apply/funds/workflow.py

```diff
@@ -115,12 +115,22 @@ def can_edit(self, user: 'User') -> bool:
         'transitions': {
             'internal_review': 'Open Review',
             'rejected': 'Reject',
+            'more_info': 'Request More Information',
         },
         'display': 'Under Discussion',
         'stage': Request,
         'permissions': Permission(),
         'step': 0,
     },
+    'more_info': {
+        'transitions': {
+            INITIAL_STATE: 'Submit',
```

Comment:
This has the interesting side effect a "Submit" action for admin in the actions to take block, which is a bit weird.

Reply:
Yeah i've been thinking about that

Permissions can be defined on the transitions:
https://github.com/kmmbvnr/django-fsm#permissions

Which can then feed into `get_available_user_FIELD_transitions` so that we can populate the the actions menu easily

Reply:
came from `attrs['restart'] = transition(attrs['status'], source='*', target=INITIAL_STATE)(lambda x: None)`
which is dev only post-fsm refactor.